### ISSUE TITLE
[DATA SCIENCE] [REMOTO] [TAMBEM PCD] CIENTISTA DE DADOS na [ZOOX]

### ISSUE THREAD

**Issue by lizvidotti91** [devssa/onde-codar-em-salvador#1214]
## Descrição da vaga

- Trabalhar com grandes volumes de dados;
- Entendimento do negócio;
- Trabalhar junto das áreas de Data Engineer e Data Analytics para
- Tratamento de datasets (Padronização, normalização, balanceamento, etc);
- Modelagem Preditiva;
- Machine Learning;
- Entendimento de arquitetura de software e programação;

## Local

- Remoto

## Benefícios

- Informações diretamente com o responsável/ recrutador da vaga

## Requisitos

**Obrigatórios:**
- Experiência com Spark, Hadoop, Pandas;
- Conhecimento em Python e R;
- Desejável conhecimento em Sistemas Linux;
- Scikit-Learn, TensorFlow, StatsModel, Keras, etc;
- Mining, Árvore de decisão, Cálculo multivariável, Álgebra Linear;
- PowerBI, Microstrategy, Tableau, etc

## Contratação

- a combinar

## Nossa empresa

- Fundada em 2010 e presente em sete países, Zoox Smart Data é uma empresa de tecnologias e soluções integradas de Big Data e Analytics que digitaliza serviços, captura dados, enriquece e entende a experiência das pessoas. Com mais de 100 colaboradores e presente em Rio de Janeiro, São Paulo, Argentina, México, Espanha, Estados Unidos e Portugal, a Zoox desponta como uma das startups mais promissoras do Brasil. Venha fazer parte do nosso time! #VoaZoox 🚀

## Como se candidatar

- [Clique aqui para se candidatar](https://zoox.gupy.io/jobs/605506?jobBoardSource=gupy_public_page)
